src/test/groovy/ee/test_gov/sid/mock/DemoMockComparisonSpec.groovy

```diff
@@ -0,0 +1,186 @@
+package ee.test_gov.sid.mock
+
+import ee.test_gov.sid.mock.data.DataProvider
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.HttpStatus
+import io.micronaut.http.MediaType
+import io.micronaut.http.client.HttpClient
+import io.micronaut.http.client.annotation.Client
+import io.micronaut.test.extensions.spock.annotation.MicronautTest
+import jakarta.inject.Inject
+import org.bouncycastle.asn1.*
+import org.bouncycastle.asn1.x500.RDN
+import org.bouncycastle.asn1.x500.X500Name
+import org.bouncycastle.asn1.x500.style.BCStyle
+import org.bouncycastle.cert.jcajce.JcaX509CertificateHolder
+import setup.RedisDependantSpecification
+import spock.lang.Ignore
+
+import java.security.cert.CertificateFactory
+import java.security.cert.X509Certificate
+import java.time.LocalDateTime
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+@Ignore("Local usage only.")
```

Comment:
Why so? Would be nice to have an extra test to add to the automated workflow.

Reply:
Considering the frequency of changes SK makes in demo certificates, this test would start failing pretty quick and therefore would start blocking dependabot PR-s. Yes, it would be nice to have the mock always exactly match demo, but since it's manual work to get new certificates for mock it's not reasonable at the time being.

Reply:
Gotcha! Makes sense to keep maintenance effort low.